docs/users-manual/job-scheduling.rst

```diff
@@ -461,3 +598,131 @@ There are some limitations to HTCondor's job deferral feature.
+by which all the different owners have graciously allowed their machine
+to be part of the HTCondor pool. You may advertise that your machine is
+only willing to run jobs at night and when there is no keyboard activity
+on your machine. In addition, you may advertise a preference (rank) for
+running jobs submitted by you or one of your co-workers.
+
+Likewise, when submitting a job, you specify a ClassAd with your
+requirements and preferences. The ClassAd includes the type of machine you wish to
+use. For instance, perhaps you are looking for the fastest floating
+point performance available. You want HTCondor to rank available
+machines based upon floating point performance. Or, perhaps you care
+only that the machine has a minimum of 128 MiB of RAM. Or, perhaps you
+will take any machine you can get! These job attributes and requirements
+are bundled up into a job ClassAd.
+
+HTCondor plays the role of a matchmaker by continuously reading all the
+job ClassAds and all the machine ClassAds, matching and ranking job ads
+with machine ads. HTCondor makes certain that all requirements in both
+ClassAds are satisfied.
+
+Inspecting Machine ClassAds with condor_status
+''''''''''''''''''''''''''''''''''''''''''''''
+
+:index:`condor_status<single: condor_status; HTCondor commands>`
+
+Once HTCondor is installed, you will get a feel for what a machine
+ClassAd does by trying the :tool:`condor_status` command. Try the
+:tool:`condor_status` command to get a summary of information from ClassAds
+about the resources available in your pool. Type :tool:`condor_status` and
+hit enter to see a summary similar to the following:
+
+.. code-block:: text
+
+    Name               OpSys      Arch   State     Activity LoadAv Mem   ActvtyTime
+
+    amul.cs.wisc.edu   LINUX      INTEL  Claimed   Busy     0.990  1896  0+00:07:04
+    slot1@amundsen.cs. LINUX      INTEL  Owner     Idle     0.000  1456  0+00:21:58
+    slot2@amundsen.cs. LINUX      INTEL  Owner     Idle     0.110  1456  0+00:21:59
+    angus.cs.wisc.edu  LINUX      INTEL  Claimed   Busy     0.940   873  0+00:02:54
+    anhai.cs.wisc.edu  LINUX      INTEL  Claimed   Busy     1.400  1896  0+00:03:03
+    apollo.cs.wisc.edu LINUX      INTEL  Unclaimed Idle     1.000  3032  0+00:00:04
+    arragon.cs.wisc.ed LINUX      INTEL  Claimed   Busy     0.980   873  0+00:04:29
+    bamba.cs.wisc.edu  LINUX      INTEL  Owner     Idle     0.040  3032 15+20:10:19
+    ...
+
+
+The :tool:`condor_status` command has options that summarize machine ads in a
+variety of ways. For example,
+
+ *condor_status -available*
+    shows only machines which are willing to run jobs now.
+ *condor_status -run*
+    shows only machines which are currently running jobs.
+ *condor_status -long*
+    lists the machine ClassAds for all machines in the pool.
+
+Refer to the :doc:`/man-pages/condor_status` command reference page for a
+complete description of the :tool:`condor_status` command.
```

Comment:
I think we can remove this given all the tool references for condor_status in the section